application/config/form_validation.php

```diff
@@ -18,63 +18,129 @@
  */
 
 // Set form validation rules here.
+
+/*form fileds:
+                //build company info array for model
```

Comment:
Codeigniter allows validation to be passed as a an array. Codeigniter knows which controller and method it applies to, when using this code: `// Set form validation rules here.
$config = array(
        // Sets validation rules for the 'Add a Gig' form
        'gig/add' => array(`
This array gets loaded automatically when the method is called.